tests/test_plot/test_map.py

```diff
@@ -35,9 +35,8 @@ def test_map_plot_with_points(tmpdir, generate_plot):
 
     mypoints = xtgeo.points_from_surface(mysurf)
 
-    df = mypoints.dataframe.copy()
-    df = df[::20]
-    mypoints.dataframe = df
+    df = mypoints.get_dataframe()
+    mypoints.set_dataframe(df[::29])
```

Comment:
Any reasons for going from `::20` to `::29`?

Reply:
Nice catch, not sure how this happened.